source/content/guides/integrated-composer/01-introduction.md

```diff
@@ -21,17 +21,17 @@ Integrated Composer is a Pantheon platform feature that extends Composer <Popove
 
 ## Get Started With Integrated Composer
 
-### Drupal 9 with Integrated Composer
+### Drupal with Integrated Composer
 
-- Follow the [Drupal 9](/drupal-9) doc to create a new Drupal 9 site with Integrated Composer built in.
+- Follow the [Drupal](/drupal) doc to create a new Drupal site with Integrated Composer built in.
```

Comment:
@stovak the first and second bullet appear to say the same thing... https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/integrated-composer